qt/scientific_interfaces/Muon/MuonAnalysis.cpp

```diff
@@ -1935,11 +1935,42 @@ void MuonAnalysis::plotSpectrum(const QString &wsName, bool logScale) {
 QMap<QString, QString> MuonAnalysis::getPlotStyleParams(const QString &wsName) {
   // Get parameter values from the options tab
   QMap<QString, QString> params = m_optionTab->parsePlotStyleParams();
+  auto upper = m_uiForm.timeAxisFinishAtInput->text().toDouble();
+
+  Workspace_sptr ws_ptr =
```

Comment:
`Workspace_const_sptr`

Reply:
this one cannot be const as it is used in a dynamic pointer cast

Reply:
The cast is to `MatrixWorkspace_sptr` which could also be `MatrixWorkspace_const_sptr`.

Reply:
yes, I now have `MatrixWorkspace_const_sptr` but line 1940 was left alone.

Reply:
You are still doing a non-`const` cast. Change the cast to `<const MatrixWorkspace>` and you can change `ws_ptr` to `Workspace_const_sptr`.